packages/viewer/src/components/scene-tree/lib/dom.ts

```diff
@@ -0,0 +1,3 @@
+export function getSceneTreeViewportHeight(el: HTMLElement): number {
+  return el.clientHeight;
+}
```

Comment:
nit: Seems like this could just be a top level `lib` rather than specific to the scene tree